src/testdrive/Cargo.toml

```diff
@@ -35,6 +35,7 @@ reqwest = { version = "0.11.0", features = ["native-tls-vendored"] }
 rusoto_core = { git = "https://github.com/rusoto/rusoto.git" }
 rusoto_credential = { git = "https://github.com/rusoto/rusoto.git" }
 rusoto_kinesis = { git = "https://github.com/rusoto/rusoto.git" }
+rusoto_s3 = { git = "https://github.com/rusoto/rusoto.git" }
 rusoto_sts = { git = "https://github.com/rusoto/rusoto.git" }
```

Comment:
Why are there multiple entries pointing at the same GitHub repo? And how is this different from something like `features = ["s3", "sts"]`?